src/sbt-test/sbt-scalafix/dependency/build.sbt

```diff
@@ -1,4 +1,4 @@
-scalaVersion := "2.12.7"
```

Comment:
Trying to keep the scalaVersion can cause errors as old semanticdb-scalac are used, for example on java11:
```
sbt:sbt-scalafix> scripted sbt-scalafix/dependency
[info] [info] Using semanticdb-scalac 4.0.0 in project dependency since the version 4.4.32 tracked by scalafix 0.9.34 is not available for scala 2.12.7 - consider upgrading sbt-scalafix or bumping scala
[info] [error] ## Exception when compiling 1 sources to /tmp/sbt_fab36d90/dependency/target/scala-2.12/classes
[info] [error] java.lang.NoClassDefFoundError: javax/xml/bind/DatatypeConverter
[info] [error] scala.meta.internal.semanticdb.scalac.InputOps$XtensionGSourceFileInput.toMD5(InputOps.scala:41)
[info] [error] scala.meta.internal.semanticdb.scalac.TextDocumentOps$XtensionCompilationUnitDocument.toTextDocument(TextDocumentOps.scala:614)
[info] [error] scala.meta.internal.semanticdb.scalac.SemanticdbPipeline$SemanticdbTyperComponent$ComputeSemanticdbPhase.saveSemanticdbForCompilationUnit(SemanticdbPipeline.scala:59)
[info] [error] scala.meta.internal.semanticdb.scalac.SemanticdbPipeline$SemanticdbTyperComponent$ComputeSemanticdbPhase.apply(SemanticdbPipeline.scala:65)
[info] [error] scala.tools.nsc.Global$GlobalPhase.applyPhase(Global.scala:441)
[info] [error] scala.tools.nsc.Global$GlobalPhase.run(Global.scala:392)
[info] [error] scala.meta.internal.semanticdb.scalac.SemanticdbPipeline$SemanticdbTyperComponent$ComputeSemanticdbPhase.run(SemanticdbPipeline.scala:75)
[info] [error] scala.tools.nsc.Global$Run.compileUnitsInternal(Global.scala:1467)
```

Reply:
This kind of regression should be mostly addressed by https://github.com/scalacenter/sbt-scalafix/pull/292/commits/b9e14bd32ee0fe18118f8bdf24b7204a77b25f60